packages/story-editor/src/components/panels/design/textStyle/textStyle.js

```diff
@@ -28,9 +28,13 @@ import PropTypes from 'prop-types';
  */
 import getUpdatedSizeAndPosition from '../../../../utils/getUpdatedSizeAndPosition';
 import { styles, useHighlights, states } from '../../../../app/highlights';
+import { useStory, useLayout } from '../../../../app';
+import { getFailingPages } from '../../../checklist/checks/pageBackgroundLowTextContrast/check';
```

Comment:
You added `getFailingPages` to exports on `pageBackgroundLowTextContrast` so you could probably import from there too.
```suggestion
import { getFailingPages } from '../../../checklist/checks/pageBackgroundLowTextContrast';
```